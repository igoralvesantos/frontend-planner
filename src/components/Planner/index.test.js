import React from "react";
import renderer from "react-test-renderer";
import { Planner } from "./index";

it("Snapshot Planner", () => {
  const tree = renderer
  .create(<Planner tasks={[]}></Planner>)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
