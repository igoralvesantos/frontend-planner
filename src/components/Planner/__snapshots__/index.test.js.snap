// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Planner 1`] = `
<div
  className="sc-htpNat cWnHDn"
>
  <div
    className="sc-bwzfXH eeNKNV"
  >
    <h3
      className="sc-bdVaJa geihCd"
    >
      Segunda
    </h3>
  </div>
  <div
    className="sc-bwzfXH eeNKNV"
  >
    <h3
      className="sc-bdVaJa geihCd"
    >
      Terça
    </h3>
  </div>
  <div
    className="sc-bwzfXH eeNKNV"
  >
    <h3
      className="sc-bdVaJa geihCd"
    >
      Quarta
    </h3>
  </div>
  <div
    className="sc-bwzfXH eeNKNV"
  >
    <h3
      className="sc-bdVaJa geihCd"
    >
      Quinta
    </h3>
  </div>
  <div
    className="sc-bwzfXH eeNKNV"
  >
    <h3
      className="sc-bdVaJa geihCd"
    >
      Sexta
    </h3>
  </div>
  <div
    className="sc-bwzfXH eeNKNV"
  >
    <h3
      className="sc-bdVaJa geihCd"
    >
      Sábado
    </h3>
  </div>
  <div
    className="sc-bwzfXH eeNKNV"
  >
    <h3
      className="sc-bdVaJa geihCd"
    >
      Domingo
    </h3>
  </div>
</div>
`;
